server/src/infra/cli.rs

```diff
@@ -94,6 +97,13 @@ pub struct TestEmailOpts {
     pub smtp_opts: SmtpOpts,
 }
 
+#[derive(Debug, Parser, Clone)]
+pub struct CreateSchemaOpts {
+    /// Database connection URL
+    #[clap(short, long, env = "LLDAP_CREATE_SCHEMA_DATABASE_URL")]
```

Comment:
I think this should just be `LLDAP_DATABASE_URL`, and we'll read from the config by default

Reply:
We could actually just make that part of the general CLI options, to be able to override the DB url on the command line.

Reply:
and you'll need to update `configuration.rs` to merge it into the config.